pom.xml

```diff
@@ -329,7 +329,6 @@ Portions Copyright (c) 2018, 2020, Chris Fraire <cfraire@me.com>.
                   <execution>
                     <goals>
                       <goal>javadoc</goal>
-                      <goal>test-javadoc</goal>
```

Comment:
Does this removal mean that Javadoc within test sources will not be checked ?

Reply:
Yes, Removed this since we were not publishing javadoc for test sources to github pages.
From junit 5 onwards, it is recommended to have default scope for all test classes and methods(instead of public scope), even for this use, test-javadoc will fail since it wont be able to findout any public java classes under test package.
Have we added test-javadoc goal considering any other usecase

Reply:
That's a good question about the use case. I am definitely not using the generated javadocs for tests so it's probably okay to say good bye to them. When changing/adding new test cases, the IDE will probably provide some sufficient warnings.